.eslintrc.js

```diff
@@ -1,46 +1,26 @@
 module.exports = {
-    env: {
-        browser: true,
-        es2021: true,
-        node: true,
-    },
-    extends: ['plugin:react/recommended', 'airbnb-typescript'],
-    parser: '@typescript-eslint/parser',
-    parserOptions: {
-        ecmaFeatures: {
-            jsx: true,
-        },
-        ecmaVersion: 12,
-        sourceType: 'module',
-        project: ['./tsconfig.json'],
-    },
-    plugins: ['react', '@typescript-eslint', 'import', 'jsx-a11y'],
+    extends: ['@prezly', '@prezly/eslint-config/react', '@prezly/eslint-config/nextjs'],
     rules: {
-        'max-len': ['warn', { code: 120 }],
-        'react/react-in-jsx-scope': 'off',
-        'react/prop-types': 'off',
-        'react/jsx-filename-extension': ['warn', { extensions: ['.tsx'] }],
-        'react/jsx-indent': ['error', 4],
-        'react/jsx-indent-props': ['error', 4],
-        'import/extensions': [
+        // TODO: NextJS Dynamic import doesn't work too well with named exports.
+        // Gotta figure out how to make them work together.
+        // Some hints here: https://github.com/vercel/next.js/issues/22278#issuecomment-1009865850
+        'import/no-default-export': 'off',
```

Comment:
I think we should follow starter theme and leave this rule enabled and only disable it explicitly in places with a comment [like here](https://github.com/prezly/theme-nextjs-starter/blob/main/modules/Search/index.ts).

Right now with this rule disabled it's pretty much a wild west in terms of default exports.

Reply:
I see that you're already doing that for certain modules anyway.

Reply:
You're right, I'll update that.

This config was pulled from Bea theme, and is already propagated to Leuven, Greta and Roompot 😕 
I hope it won't bite us in the long run